core.js

```diff
@@ -217,6 +217,7 @@ function processItemAsync(runner, currentValidation, key, context) {
 function addError(errors, key, validation) {
   return function(err) {
     var fieldError = errors[key];
+    err.message = validation.message || err.message;
```

Comment:
Yes, you're right. I tried another way like this:
``` javascript
var errorMessage = validation.message || err.message;

...

fieldError = errors[key] = new FieldError(errorMessage)
```

But unfortunately that only works for one validation-rule per field, as it will be only applied if `fieldError` is still undefined. If there are several errors for one field, the error-objects will just be pushed to the array `fieldError.errors` (in line 226).